### PR TITLE
Fixes #39: Fixed test server startup timeouts.

### LINKED ISSUE #39: Wait for test server to be started before proceeding further
See https://github.com/Kinto/kinto-client/pull/38#issuecomment-190610400

### PR DESCRIPTION
**Pull request by n1k0**
Refs #39. r=? @almet

### CODE REVIEW

**Comment by n1k0** [comment]
You know what? I'm landing this.

**Comment by almet** [comment]
That's really cool, we should maybe try to do the same for all the requests?

File: test/server_utils.js [hunk 1]
```diff
@@ -15,33 +15,61 @@ export default class KintoServer {
     this.options = Object.assign({}, DEFAULT_OPTIONS, options);
   }
 
+  _retryRequest(url, options, attempt=1) {
+    const { maxAttempts } = this.options;
+    return fetch(url, options)
+      .then(res => {
+        if ([200, 202, 410].indexOf(res.status) === -1) {
```
Comment (Natim) [inline test/server_utils.js]: Are you sure about 202? shouldn't it be 201?

Comment (n1k0) [inline test/server_utils.js]: Flush returns a 202.

**Pull request merged**